components/NavBar/index.js

```diff
@@ -151,25 +153,30 @@ const NavBar = () => {
                 </button>
               </a>
             </Link>
+
```

Comment:
do we need this extra space?

Reply:
Will remove it in the upcoming commit 👍🏻